### PR TITLE
Kubernetes worker pool config

### PR DESCRIPTION
**Pull request by rtannenbaum**
Workers are up on our QA instance and are polling, but encryption flow in API is not enabled.

Also switch logging level for QA to DEBUG so we can see the worker polling statements in QA.

for #88

### CODE REVIEW

File: config/k8s/worker-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /var/secrets/portability-service-account-key/key.json
+        - name: GOOGLE_PROJECT_ID
+          value: world-takeout-qa
+        resources: {}
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
```
Comment (holachuy) [inline config/k8s/worker-deployment.yaml]: Is this what makes it restart when it fails?

File: config/k8s/worker-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
```
Comment (holachuy) [inline config/k8s/worker-deployment.yaml]: Let's add the license header and also an description including that we count on these to restart themeselves.

File: config/gcp/setup_gke_environment.sh [hunk 3]
```diff
@@ -54,6 +55,111 @@ print_step() {
...
+
+  print_step "Importing the service account credentials, created earlier, as a Kubernetes Secret"
+  kubectl create secret generic portability-service-account-creds --from-file=key.json=/tmp/service_account_creds.json
+
+  if [[ ${BACKEND} != "api" ]]; then
+    echo -e "Done creating ${BACKEND} pool!"
+    return
+  fi
+
+  # -----------------------------------------------------------------------
```
Comment (holachuy) [inline config/gcp/setup_gke_environment.sh]: Should this be a separate method?

**Comment by rtannenbaum** [comment]
I tried rebasing but I think it's too late since I had done some merges. I'll stick with rebasing from now on.

**Comment by holachuy** [comment]
Yeah, I still don't know what works best as far as a git workflow but it usually leaves the history a bit cleaner. Maybe you can squash and merge in as one commit to avoid to long commit history

**Pull request merged**